client/packages/common/src/intl/locales/en/coldchain.json

```diff
@@ -20,6 +20,7 @@
   "label.calculate-demand": "Enabled",
   "label.day": "day",
   "label.target-demographic": "Target Demographic",
+  "label.new-row": "new row",
```

Comment:
I think remove this one? You also added it in `common` with the capitals, which I think looks nicer, but I'm still getting this one in UI:

![Screenshot 2024-06-17 at 3 05 58 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/dc087c22-8630-42c4-9b9f-7a22fd20dd93)